src/Widgets/calendar/ControlHeader.vala

```diff
@@ -19,35 +19,85 @@
 
 
 namespace DateTime.Widgets {
-    public class ControlHeader : Gtk.Box {
+    public class ControlHeader : Gtk.Grid {
         public signal void left_clicked ();
         public signal void right_clicked ();
         public signal void center_clicked ();
+
         public ControlHeader () {
             Object (orientation : Gtk.Orientation.HORIZONTAL);
+        }
+
+        construct {
+            string style = """
+                .header-label {
+                    font-family: "Raleway", sans;
+                    font-weight: 300;
+                    font-size: 1.66em;
+                }
+            """;
+
+            var provider = new Gtk.CssProvider ();
+            try {
+                provider.load_from_data (style, style.length);
+                Gtk.StyleContext.add_provider_for_screen (Gdk.Screen.get_default (), provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
+            } catch (Error e) {
+                critical (e.message);
+            }
+
+            var label = new Gtk.Label (new GLib.DateTime.now_local ().format (_("%OB, %Y")));
+            label.get_style_context ().add_class ("header-label");
+            label.xalign = 0;
+            label.width_chars = 13;
+
             var left_button = new Gtk.Button.from_icon_name ("pan-start-symbolic");
+            var center_button = new Gtk.Button.from_icon_name ("office-calendar-symbolic");
+            center_button.tooltip_text = _("Go to today's date");
             var right_button = new Gtk.Button.from_icon_name ("pan-end-symbolic");
-            var center_button = new Gtk.Button.with_label (new GLib.DateTime.now_local ().format (_("%OB %Y")));
+
+            var box_label = new Gtk.Grid ();
```

Comment:
Why are you adding a new grid that only contains one object?

Reply:
It was part of a dirty hack involving the dummy spacer, but your PR at https://github.com/lainsce/wingpanel-indicator-datetime/pull/4 is a smarter approach.